libmisc/copydir.c

```diff
@@ -428,16 +428,16 @@ static int copy_entry (const struct path_info *src, const struct path_info *dst,
 		mt[1].tv_nsec = sb.st_mtim.tv_nsec;
 
 		if (S_ISDIR (sb.st_mode)) {
-			err = copy_dir (src, dst, reset_selinux, &sb, mt,
-			                old_uid, new_uid, old_gid, new_gid);
+			err_dir = copy_dir (src, dst, reset_selinux, &sb, mt,
+			                 old_uid, new_uid, old_gid, new_gid);
```

Comment:
This line shouldn't be touched if not needed. And if you want to update indentation you should update it properly.